### PR TITLE
Ds/feature/map

### LINKED ISSUE #98: Add map base to styles page
Add the correct mapping style that should be used across WMN services.
This should be the BCCI mapping or as close to as possible.

Provide screenshots of what it should look like.

Relates to: https://github.com/wmcadigital/wmn-sprints/issues/21

### PR DESCRIPTION
**Pull request by daylesalmon**
Fixes #98

### CODE REVIEW

File: src/www/pages/styles/maps/index.njk [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+      linkTarget: "_blank",
+      text: "ArcGIS APIs and SDKs"
+    })
+  }}.
+</p>
+
+{# Basemap #}
+<h3>Basemap</h3>
+  <a href="//disruptions.wmnetwork.co.uk/" class="wmnds-col-1" target="_blank" rel="noopener noreferrer">
+    <img class="wmnds-col-1 wmnds-m-b-md" src="$*cdn/img/component-images/maps/base-map.png" alt="A screenshot showing an example of a map">
```
Comment (catiarodriguescosta) [inline src/www/pages/styles/maps/index.njk:38]: This image it's not loading in the preview page, is that okay? Or does source has any typo? (not sure how cdn works here)

Comment (daylesalmon) [reply src/www/pages/styles/maps/index.njk]: @catiarodriguescosta This should be resolved in my last commit.

**Review by gldgrnt — APPROVED** [review]
All good with me

**Review by catiarodriguescosta — APPROVED** [review]

**Pull request merged**